src/client/datascience/variablesView/notebookWatcher.ts

```diff
@@ -8,13 +8,13 @@ import { IFileSystem } from '../../common/platform/types';
 import { IDisposableRegistry } from '../../common/types';
 import { KernelState, KernelStateEventArgs } from '../notebookExtensibility';
 import { INotebook, INotebookEditor, INotebookEditorProvider, INotebookExtensibility } from '../types';
-import { IVariableViewNotebookWatcher } from './types';
+import { INotebookWatcher } from './types';
```

Comment:
Should the path be different if not specific to variables? It's only used for variables right now, so maybe not.